fastapi/environment.yml

```diff
@@ -0,0 +1,8 @@
+channels:
+  - conda-forge
+  
+dependencies:
+  - fastapi
+  - pydantic==2.6.3
```

Comment:
not saying it's bad, but is there a reason why pydantic is pinned?

Reply:
It's been a while so I may not remember exactly what change was needed but the `pydantic` dep version was different when setting up the conda env on my personal machine vs the EC2. This caused some model errors on the EC2 instance that I wasn't seeing in my local instance. I found that I needed v2.6.3 to support the models as I have them currently, so I figured I would pin the version so I wouldn't run into the same issue again.